core/shared/src/main/scala/fs2/Collector.scala

```diff
@@ -83,12 +83,12 @@ object Collector extends CollectorPlatform {
     def array[A: ClassTag]: Builder[A, Array[A]] =
       Chunk.newBuilder.mapResult(_.toArray)
 
-    protected def fromBuilder[A, C[_], B](
-        builder: collection.mutable.Builder[A, C[B]]
-    ): Builder[A, C[B]] =
-      new Builder[A, C[B]] {
+    protected def fromBuilder[A, C](
```

Comment:
I hope this change makes sense and doesn't break binary compatibility.
Although, the code compiled without it so we revert it if you prefer.